eks_addon/run_tests.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Tests the Splunk OpenTelemetry Collector EKS Add-on for compliance with AWS requirements.
+# This script prepares the chart build and runs tests to verify it meets EKS addon requirements.
+
+# Enable bash strict mode
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPT_DIR}/common.sh"
+
+if [[ -z "${K8S_VERSION}" ]]; then
+  echo "❌ Error: K8S_VERSION environment variable is required"
+  exit 1
+fi
+
+# EKS Add-on specific test functions
+test_helm_lint() {
+  echo "⏳ Running helm lint on ${EKS_CHART_DIR} ..."
```

Comment:
I wonder if it would be helpful at the beginning of each test to also log the function name for easier searching when they fail. I find it a lot easier to search function names in repos vs. plaintext.

Reply:
I simplified the logs to include the test name only